packages/api-explorer/src/lib/Oas.js

```diff
@@ -29,7 +29,8 @@ class Operation {
         return keys.map(key => {
           let security;
           try {
-            security = this.oas.components.securitySchemes[key];
+            // old version of swagger has this.securitySchemes instead of this.securityDefinitions
```

Comment:
We support both swagger and openapi3 format? As I know we support only OAS3, but I'm not sure

Reply:
I think so, previous implementation only used `securitySchemes` which is part of the swagger definition while openapi v3 uses `securityDefitinions`